### DISCUSSION TITLE
Why is there no compatibility for CTkEntry?

### DISCUSSION THREAD

**Discussion by dogabayam** [TomSchimansky/CustomTkinter#397] [Q&A]
Why cant we use tk.Entry arguments and methods with CTKEntry like most other widgets included in customtkinter?

**Comment by TomSchimansky** [comment]
Which arguments and methods are you missing?

**Comment by dogabayam** [comment]
Sorry for the lack of content and generalization in my first post. I was specifically talking about methods like **index** and **icursor** from tk.Entry. I am trying to create an autocomplete entry widget based on CTkEntry however this object lacks the methods I mentioned and makes it difficult to handle some aspects. Am I missing something?
